acme/acme/jose/jwk.py

```diff
@@ -174,9 +174,10 @@ def public_key(self):
 class JWKRSA(JWK):
     """RSA JWK.
 
-    :ivar key: `cryptography.hazmat.primitives.rsa.RSAPrivateKey`
+    :ivar key: Optional `cryptography.hazmat.primitives.rsa.RSAPrivateKey`
```

Comment:
nit: This documentation is for the `key` instance variable of a `JWKRSA` object and it doesn't really make sense to call it optional. To document you don't have to pass a key to the constructor, you should either add documentation to the `__init__` method or remove "Optional" at the beginning and make the behavior clear in the other sentence you added here.